### PR TITLE
help page

### PR DESCRIPTION
**Pull request by vsrisuknimit**
What I did:
- Add a page that show game iconography

image

What I would like to do next:
- Add a link to the side bar. A question mark button. Put it at the very bottom but above the gear. 
- Click on that will link to this page
- Make the global parameter center aligned.
- Fix this page in general cause I don't know what I am doing.

This will partially address #1457 .

### CODE REVIEW

File: src/components/HelpIconology.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+export const HelpIconology = Vue.component("help-iconology", {
+    components: {
+        Card
+    },
+    methods: {
+        
+    },
+    template: `
+        <div class="help-iconology-container">
+            <div class="row" style="display: flex;">
```
Comment (kberg) [inline src/components/HelpIconology.ts:13]: How comfortable are you with defining CSS and classes? If you're not, I can step in. What I'd suggset is adding help-iconology.less in src/styles with this content

.help-iconology-container .row {
  display: flex;
}

.help-iconology-container .column {
  flex: 25%;
padding: 10px;
}

File: src/components/HelpIconology.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                <div class="column" style="flex: 25%; padding: 10px;">
+                    <section class="help-iconology-tags">
+                    <h2>Card Tags</h2>
+                        <div class="resource-tag tag-building" /> : Building tag<br>
```
Comment (kberg) [inline src/components/HelpIconology.ts:17]: No need to add the word "tag" after each as it's obvious they're tags, being in the tags section.

**Review by kberg — CHANGES_REQUESTED** [review]
LMK if you have questions.

**Comment by kberg** [comment]
Oh also!

This is very helpful -- thank you! I could see adding this off the player's page with an <a target="_blank"> tag.

File: src/components/HelpIconology.ts (hunk 1 above)
Comment (vsrisuknimit) [reply src/components/HelpIconology.ts:13]: Moved

File: src/components/HelpIconology.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                        <div class="resource-tag tag-animal" /> : Animal tag<br>
+                        <div class="resource-tag tag-city" /> : City tag<br>
+                        <div class="resource-tag tag-wild" /> : Wild tag<br>
+                        <div class="resource-tag tag-event" /> : Event tag<br>
+                    </section>
+                </div>
+                <div class="column" style="flex: 25%; padding: 10px;">                        
+                    <section class="help-iconology-resources">
+                    <h2>Resources</h2>
+                        <div class="resource money" /> : Mega credits<br>
```
[the remark this answers is not in the document]
Comment (vsrisuknimit) [reply src/components/HelpIconology.ts]: fixed

File: src/components/HelpIconology.ts (hunk 2 above)
Comment (vsrisuknimit) [reply src/components/HelpIconology.ts]: deleted

**Review by kberg — APPROVED** [review]

**Comment by kberg** [comment]
This is terrific. Thank you!

**Comment by kevinb9n** [comment]
Yes, this is really fantastic!

I have small suggestions. Sorry for picking, I really do appreciate this. Please ignore any of the below that you don't personally find helpful.

First some slight corrections:

* It's just "terraform rating".
* "Megacredits" is one word. The rules sometimes camel-case it "MegaCredits" but not always. Perhaps put "(M€)" after it.
* TR/Colony/TradeFleet need some heading of their own ("Other"?), as they're not resources.
* Visual: dunno if you noticed the Greenery tile is randomly wider than the others.

Possible additions - just ignore the ones you find unnecessary?

* Project card (seen on map and some cards)
* Delegate
* Victory Point should be in here using whatever style is currently in use.
* Perhaps a hollow production box can be shown, labeled as "Production"?

Suggested changes:

* I think you can omit the Fighter icon for the same reason you're omitting Camp and Preservation resources - they're single-use.
* Separate sections labeled "Standard Resources" and "Card-based Resources" would be helpful (e.g. for PhilAres).
* I would like it very much if we could set "Wild tag" and "Event tag" apart from the rest somehow, even if just by a little extra space, as it is sometimes important to understand that they *are not tags*. I know I am a little more bothered by this than most, but I actually always put "tag" in quote marks when talking about these, or use the phrase "Event symbol"/"Wild symbol".
* Among the 11 actual tags, it would also be useful to set Venus apart a bit, even if not actually labeling it "(expansion only)". (I'd still keep Earth and Jovian just before it.) The effect of this and the previous suggestion is to make it easier to understand why the most M€ production you can ever get from Interplanetary Trade in a no-expansions game is 10. That's _very_ nonobvious as things stand.
* SUPER NITPICK ALERT, under global parameters, it's a little better to say "Oceans" with an s.

**Comment by kevinb9n** [comment]
Important addendum: if you prefer, please feel free to just tell me which ones you disagree with and wait for me to try hacking up the file myself.

**Pull request merged**